analyser_merge_milestone_FR_metropole.py

```diff
@@ -0,0 +1,127 @@
+    def __init__(self, config, logger = None):
+        Analyser_Merge.__init__(self, config, logger)
+
+        doc = dict(
+            detail = T_(
+            '''A road milestone is present but not integrated. The list of milestone comes from the CEREMA's database "RIU" in France by the Ministry of the Ecological and Inclusive Transition.'''),
+            fix = T_(
+            '''Add a node or to integrate tags if milestone already existing.'''),
+            trap = T_(
+            '''The position of the marker may be a little different than that visible on the road. Sometimes, a small white line perpendicular to the road on the emergency stop strip or the left flared strip can be seen on satellite images or mapillary's photo
+            '''))
+
+        self.def_class_missing_official(item = 8130, id = 41, level = 3, tags = ['merge', 'highway'],
+            title = T_('Milestone not integrated'), **doc)
+        self.def_class_possible_merge(item = 8131, id = 43, level = 3, tags = ['merge', 'highway'],
+            title = T_('Milestone integration suggestion'), **doc)
+        self.def_class_update_official(item = 8132, id = 44, level = 3, tags = ['merge', 'highway'],
+            title = T_('Milestone update'), **doc)
+
+        self.init(
+            u"https://www.data.gouv.fr/fr/datasets/bornage-du-reseau-routier-national/",
+            u"Bornage du réseau routier national",
+            CSV(Source(u"data.gouv.fr:Ministère de la Transition écologique et solidaire", millesime = "01/2019",
+                    fileUrl = u"https://www.data.gouv.fr/fr/datasets/r/fbc8b73b-a65c-486b-a710-ed22b9e4070c",
+                    separator = "\t"),
+                    
+            Load("x", "y",srid = 2154,
+                xFunction = self.float_comma, 
+                yFunction = self.float_comma,
+                where = lambda row: self.is_natref(row)),
+            Mapping(
+                select = Select(
+                    types = ["nodes"],
+                    tags = [{"highway": "milestone"}]),
+                osmRef = "nat_ref",
+                extraJoin = "ref",
+                conflationDistance = 150,
+                generate = Generate(
+                    static1 = {"highway": "milestone"},
+                    static2 = {"source": self.source},
+                    mapping1 = {
+                        "distance": 'pr',
+                        "nat_ref": lambda row: self.transform_to_natref(row),
+                        "ref": lambda row: self.transform_route(row['route']) }
+                    )))
+    
+    def is_natref(self,row):
+        if (len(row['depPr']) == 3) :
+            return False
+        elif (row['route'][0:1] == 'P') :
+            #P for temporary,
+            return False
+        elif (row['route'][0:2] in ('N1','N2')) :
+            #N1 for future up_class and N2 for down_class road,
+            return False
+        elif (row['route'][2:4] in ('A9','N9')) :
```

Comment:
Just for code style.

```suggestion
        if len(row['depPr']) == 3:
            return False
        elif row['route'][0:1] == 'P':
            #P for temporary,
            return False
        elif row['route'][0:2] in ('N1', 'N2'):
            #N1 for future up_class and N2 for down_class road,
            return False
        elif row['route'][2:4] in ('A9', 'N9'):
```